src/main/java/edu/cornell/kfs/module/purap/service/impl/CuElectronicInvoiceHelperServiceImpl.java

```diff
@@ -1289,6 +1283,16 @@ protected void rejectElectronicInvoiceFile(ElectronicInvoiceLoad eInvoiceLoad, S
         }
     }
 
+    private void clearErrorMessagesToPreventRiceSaveAndRoutErrors() {
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("clearErrorMessagesToPreventRiceSaveAndRoutErrors, Logging previously created error messages");
+            for (String key : GlobalVariables.getMessageMap().getErrorMessages().keySet()) {
+                LOG.debug("clearErrorMessagesToPreventRiceSaveAndRoutErrors, key: " + key + "value: " + GlobalVariables.getMessageMap().getErrorMessages().get(key));
```

Comment:
In the Map returned by MessageMap.getErrorMessages(), the elements appear to be of type AutoPopulatingList. This particular List implementation doesn't appear to contain the needed toString() override for printing the list contents, so if you indeed want to log the contents here, you may need additional code to implement another way of printing them.

Reply:
thanks for noticing, I will see if I can adjust the printout without significant effort, if not, I will remove this.